### PR TITLE
Mention histograms in doc.go.

### PR DESCRIPTION
**Pull request by juliusv**

### CODE REVIEW

**Comment by juliusv** [comment]
@beorn7

File: prometheus/doc.go [hunk 1]
```diff
@@ -61,12 +61,13 @@
 // It also exports some stats about the HTTP usage of the /metrics
 // endpoint. (See the Handler function for more detail.)
 //
-// A more advanced metric type is the Summary.
+// Two more advanced metric types are the Summary and Histogram types.
```
Comment (brian-brazil) [inline prometheus/doc.go]: I'd remove the second 'types'

**Comment by brian-brazil** [comment]
:+1:

File: prometheus/doc.go (hunk 1 above)
Comment (juliusv) [inline prometheus/doc.go]: Done.

**Pull request merged**